WooCommerce/Classes/ViewModels/Order Details/OrderDetailsNotices.swift

```diff
@@ -4,22 +4,50 @@ import Yosemite
 final class OrderDetailsNotices {
     /// Displays the `Unable to delete tracking` Notice.
     ///
-    func displayDeleteErrorNotice(order: Order, tracking: ShipmentTracking, onAction: @escaping () -> Void) {
-        let titleFormat = NSLocalizedString(
-            "Unable to delete tracking for order #%1$d",
-            comment: "Content of error presented when Delete Shipment Tracking Action Failed. "
-                + "It reads: Unable to delete tracking for order #{order number}. "
-                + "Parameters: %1$d - order number"
-        )
-        let title = String.localizedStringWithFormat(titleFormat, order.orderID)
-        let actionTitle = NSLocalizedString("Retry", comment: "Retry Action")
+    func displayDeleteTrackingErrorNotice(order: Order, tracking: ShipmentTracking, onAction: @escaping () -> Void) {
+        let title = String.localizedStringWithFormat(Localization.deleteTracking, order.orderID)
         let notice = Notice(title: title,
                             message: nil,
                             feedbackType: .error,
-                            actionTitle: actionTitle) {
+                            actionTitle: Localization.retry) {
                                 onAction()
         }
 
         ServiceLocator.noticePresenter.enqueue(notice: notice)
     }
+
+    /// Displays the `Unable to trash order` Notice.
+    ///
+    func displayTrashOrderErrorNotice(order: Order, onAction: @escaping () -> Void) {
+        let title = String.localizedStringWithFormat(Localization.trashOrder, order.orderID)
+        let notice = Notice(title: title,
+                            message: nil,
+                            feedbackType: .error,
+                            actionTitle: Localization.retry) {
+                                onAction()
+        }
+
+        ServiceLocator.noticePresenter.enqueue(notice: notice)
+    }
+}
+
+private extension OrderDetailsNotices {
+    enum Localization {
+        // Here to be added to Localizable.strings so it can be looked up by the `LocalizedStringResource` above
+        static let deleteTracking = NSLocalizedString(
+                     "OrderDetail.deleteTracking.notice.title",
```

Comment:
The same applies to other strings in this file as well. 
```suggestion
                     "orderDetail.deleteTracking.notice.title",
```